Makefile

```diff
@@ -2,6 +2,7 @@
 .PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch lint fix test run dev prod
 
 SHELL = /usr/bin/env bash
+MAKEFLAGS += --jobs=4
```

Comment:
this is an enforcement rather than a default. Haven't found a sane way to allow override with `--jobs` yet.

Reply:
This puts it at the end? If you can put it at beginning, the last value is used (ie this becomes a default).

Reply:
Can do this with
```
MAKEFLAGS ::= "-j 4 $(MAKEFLAGS)" 
```
but `MAKEFLAGS` is a magic variable so it gets rewritten to `-j --jobserver-fds=3,4` which I believe is what should be passed to recursive `$(MAKE)` invocations.

Reply:
So current status of this PR is to enforce the parallelism, but making sure that targets are never in a race condition. This should cater to maximum performance, but the number of total jobs is not customizable (we could use load average as the limit rather than jobs count, but the builds are not so long that it would be a useful metric).